pit30m/data/partitions.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+from abc import abstractmethod
+from enum import Enum
+
+import numpy as np
+
+
+class PartitionEnum(Enum):
```

Comment:
Nit: Unless you have a strong preference, it's fine to not name this `...Enum`

Reply:
Good point! Addressed in [e92f984](https://github.com/pit30m/pit30m/pull/35/commits/e92f9842707ef920ff901e48725ef0249c958608)